webpack/localization-plugin/src/LocalizationPlugin.ts

```diff
@@ -655,4 +724,41 @@ export class LocalizationPlugin implements Webpack.Plugin {
       suffix: suffix
     };
   }
+
+  private _chunkHasLocalizedModules(chunk: Webpack.compilation.Chunk): boolean {
+    if (EntityMarker.getMark(chunk) === undefined) {
+      let chunkHasAnyLocModules: boolean = false;
+      if (!chunkHasAnyLocModules) {
+        for (const module of chunk.getModules()) {
+          if (EntityMarker.getMark(module)) {
+            chunkHasAnyLocModules = true;
+            break;
+          }
+        }
+      }
+
+      // Check async chunks if this is a runtime chunk and we haven't directly found any localized modules
+      if (chunk.hasRuntime() && !chunkHasAnyLocModules) {
+        for (const asyncChunk of chunk.getAllAsyncChunks()) {
+          if (this._chunkHasLocalizedModules(asyncChunk)) {
+            chunkHasAnyLocModules = true;
+            break;
+          }
+        }
+      }
+
+      EntityMarker.markEntity(chunk, chunkHasAnyLocModules);
+    }
+
+    return EntityMarker.getMark(chunk)!;
+  }
+
+  private _convertLocalizationFileToLocData(locFile: ILocalizationFile): ILocaleFileData {
+    const locFileData: ILocaleFileData = {};
+    for (const stringName in locFile) { // eslint-disable-line guard-for-in
```

Comment:
`for (const [stringName, { value }] of Object.entries(locFile))` avoids the eslint issue